src/components/view/DetailsTab.vue

```diff
@@ -29,17 +29,19 @@
       </div>
     </a-list-item>
     <DedicateData :resource="resource" v-if="dedicatedSectionActive" />
+    <VMWare :resource="resource" v-if="this.$route.meta.name === 'zone'" />
```

Comment:
@RitchieVincent in template, variables can be accessed without using `this.`.

Reply:
Also, didn't we discuss to include VMware section as a separate Tab in the zone resource view. That means not embedded within the detailstab component, but as a separate tab component.